<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1359</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>9</y>
      <width>1091</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pos">
     <attribute name="title">
      <string>Pos</string>
     </attribute>
     <widget class="QCustomPlot" name="graph_pos" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>1081</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graph_vel" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1081</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <zorder>graph_vel</zorder>
     <zorder>graph_pos</zorder>
    </widget>
    <widget class="QWidget" name="vel">
     <attribute name="title">
      <string>Vel</string>
     </attribute>
     <widget class="QCustomPlot" name="graph_vel_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1081</width>
        <height>671</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="connect_butt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: qlineargradient(spread:pad, x1:0.034, y1:0.988409, x2:0, y2:1, stop:0 rgba(0, 241, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>    COM  :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_butt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_butt">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="message_txt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>527</y>
      <width>240</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="run_butt">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveGraph_butt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Graph</string>
    </property>
   </widget>
   <widget class="QComboBox" name="port_cb">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select COM</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="ref_pos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>240</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>192</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Kp</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Ki</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Kd</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Set Point</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="Kp_edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="Ki_edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="Kd_edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="SetP_edit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="send_butt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Xmessage_butt">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>528</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="X_ref_pos_butt">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>341</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <zorder>message_txt</zorder>
   <zorder>tabWidget</zorder>
   <zorder>connect_butt</zorder>
   <zorder>label_11</zorder>
   <zorder>clear_butt</zorder>
   <zorder>refresh_butt</zorder>
   <zorder>run_butt</zorder>
   <zorder>saveGraph_butt</zorder>
   <zorder>port_cb</zorder>
   <zorder>ref_pos</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>send_butt</zorder>
   <zorder>Xmessage_butt</zorder>
   <zorder>X_ref_pos_butt</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1359</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
